extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -519,6 +502,35 @@ export class Bind {
   }
 
   /**
+   * Observes the dynamic children of `element` for mutations, if any,
+   * for rescanning for bindable attributes.
+   * @param {!Element} element
+   * @private
+   */
+  observeDynamicChildrenOf_(element) {
+    if (typeof element.getDynamicElementContainers === 'function') {
+      element.getDynamicElementContainers().forEach(this.boundObserveElement_);
```

Comment:
You could also have done `forEach(this.observeElement_, this)`, but this works too.